adafruit_lsm6ds/lsm6ds3.py

```diff
@@ -0,0 +1,45 @@
+# SPDX-FileCopyrightText: Copyright (c) 2020 Bryan Siepert for Adafruit Industries
```

Comment:
You can put your own name, and the current year for this copyright comment in the new file.